src/components/PostDetails.tsx

```diff
@@ -1,106 +1,56 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from './Loader';
-import { NewCommentForm } from './NewCommentForm';
+
+import { useAppContext } from '../context/store';
+import { Notification } from './Notification';
+import { Error } from '../types/Notification';
+import { Comments } from './Comments';
+import { getSelectedPostComments } from '../api-services/comments';
 
 export const PostDetails: React.FC = () => {
+  const {
+    state: { selectedPost },
+    methods: { setComments },
+  } = useAppContext();
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState('');
+
+  useEffect(() => {
+    if (selectedPost) {
+      setLoading(true);
+
+      getSelectedPostComments(selectedPost.id)
+        .then(fetchedComments => {
+          setError('');
+          setComments(fetchedComments);
+        })
+        .catch(() => {
+          setError(Error.CommentsError);
+        })
+        .finally(() => setLoading(false));
+    }
+  }, [selectedPost]);
```

Comment:
<img width="936" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-posts/assets/64750958/5547348d-0ab9-4686-8240-0b4c0cb2eb23">

Reply:
The same here

Reply:
Do not know what both VS code suggestions are for.